codechecker_common/plist_parser.py

```diff
@@ -301,6 +303,9 @@ def remove_report_from_plist(plist_file_obj, skip_handler):
     report_data = None
     try:
         report_data = parse_plist(plist_file_obj)
+    except plistlib.InvalidFileException as ifex:
```

Comment:
You do not use `ifex` so it can be removed.

```suggestion
    except plistlib.InvalidFileException:
```